services/speech_to_text/v1.js

```diff
@@ -85,20 +98,37 @@ module.exports = function (RED) {
 
       var s2t = function (audio, format, cb) {
         if (format !== 'wav' && format !== 'flac' && format !== 'ogg') {
-          var message = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
-          node.error(message, msg)
+          var message_err_format = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
+
+          node.error(message_err_format, msg)
           return;
         }
 
-        if (format === 'ogg') format += ';codecs=opus';
+        if (format === 'ogg') { 
+          format += ';codecs=opus';
+        }
+
+        if(config.continuous === 'true')
+        {
+          var params = {
+            audio: audio,
+            content_type: 'audio/' + format,
+            model: model,
+            continuous: true
+          };
+
+        } else {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["params" is already defined](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2398740748/issues/source?bid=3225115&fileBranchId=3231190#l121)